Psychologist Loadout Improvements

alterae opened the pull request:
## About the PR

Adds three new alternative job title PDAs:
- Psychiatrist
- Therapist (req. 4hrs psych)
- Social Worker (req. 4hrs service)

Adds head, outer clothing, scarf, and shoe loadout groups to the psychologist.

Adds a blue folder to the psychologist's starting equipment.

## Why / Balance

This change adds a bit more variety to the psychologist's starting loadout options. Since psychologist is an RP-heavy role, this allows for a bit more diversity of RP and appearances, especially with the new alternative job titles. Also, the capacity to spawn with a coat is invaluable on Glacier.

For more discussion, see [the initial WYCI suggestion](https://discord.com/channels/968983104247185448/1318589688398741574https://discord.com/channels/968983104247185448/1318589688398741574) on the DeltaV discord.

## Technical details

Adds a handful of new loadouts and new loadout groups, and makes the YAML changes necessary to have them work. Also adds three new sprites and a few new localizations for loadout group names and alternative job titles. No C# changes whatsoever.

## Media

<details><summary>in-game screenshots</summary>
<img width="402" alt="image" src="https://github.com/user-attachments/assets/818ef3b5-7d1f-41c1-aeb5-b8f802ba57b8" />
<img width="488" alt="image" src="https://github.com/user-attachments/assets/42f2bdd7-870c-4294-855e-3704b55c9b1f" />
<img width="416" alt="image" src="https://github.com/user-attachments/assets/7c517cd9-53df-4395-8d0d-1e82756953c6" />
<img width="311" alt="image" src="https://github.com/user-attachments/assets/0683399c-1565-41e7-8f71-dfae1382a862" />
<img width="467" alt="image" src="https://github.com/user-attachments/assets/9b0693e1-b901-453c-a3af-3d167d7c6d22" />
</details>

<details><summary>loadout menu screenshots</summary>
<img width="1025" alt="image" src="https://github.com/user-attachments/assets/76aa46cf-f427-4426-9cad-ed5ecce65e3a" />
<img width="694" alt="image" src="https://github.com/user-attachments/assets/d3a441e9-d125-4327-ae58-cb8f66b120bc" />
<img width="692" alt="image" src="https://github.com/user-attachments/assets/efdb50ca-02bd-40cc-a9d7-9df7dcdbc361" />
<img width="696" alt="image" src="https://github.com/user-attachments/assets/f017a312-5f54-4b70-bd35-ff3cb490beff" />
<img width="697" alt="image" src="https://github.com/user-attachments/assets/2e3fa575-2e28-4662-8e66-9c1b4123cdf2" />
</details>

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

N/A.

**Changelog**

:cl:
- add: Added coats, hats, scarves, and shoes to the Psychologist's loadout options.
- add: Added three new Psychologist job titles.
- tweak: Psychologist now spawns with a folder.

alterae committed:
Add new sprites for alternative psychologist PDAs

alterae committed:
Add localizations for alternative psychologist job names

alterae committed:
Add localizations for additional psychologist loadout groups

alterae committed:
Expand psychologist loadout options

Adds three new alternative job title PDAs:
- Psychiatrist
- Therapist (req. 4hrs psych)
- Social Worker (req. 4hrs service)

Adds head, outer clothing, scarf, and shoe loadout groups to the psychologist.

Adds a blue folder to the psychologist's starting equipment.

alterae committed:
Merge branch 'DeltaV-Station:master' into psychology-loadouts

Lyndomen commented on Resources/Prototypes/Loadouts/role_loadouts.yml:
Untouch line 389

alterae replied:
ough i thought i did hang on

Lyndomen commented on Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml:
Untouch whitespace 231 & 260

alterae committed:
Restore erroneous whitespace

I hate YAML.
This whitespace should not exist, but removing it would cause merge conflicts or something equally stupid.

Signed-off-by: Michelle Szinger <chell@alterae.online>

alterae committed:
Restore erroneous trailing whitespace

I hate YAML.
This whitespace should not exist, but its removal would lead to merge conflicts or diff messiness or other such nonsense, so.

Signed-off-by: Michelle Szinger <chell@alterae.online>

alterae committed:
Remove erroneously restored erroneous whitespace

This one is actually my bad. Apologies.

Signed-off-by: Michelle Szinger <chell@alterae.online>

alterae commented:
Apologies for doing the whitespace fixup in the web UI, but it was literally IMPOSSIBLE to do in Rider for some reason (it either would not let me commit the fix or was un-fixing it before it could be committed).

All the actual meaningful code changes were done in Rider, though, and were verified to be functional (at least on my machine).

Radezolid commented on Resources/Prototypes/Loadouts/role_loadouts.yml:
Not a big deal but in the future you might want to, instead of adding a comment in each line, bundle them inside two comments being like:
```
# Begin DeltaV additions - What you added
- blah blah
- blah blah
# End DeltaV additions
```

alterae replied:
Noted! I do think this potentially makes for cleaner diffs? But noted!

alterae committed:
Merge branch 'master' into psychology-loadouts

alterae committed:
Merge branch 'master' into psychology-loadouts

deltanedas commented on Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml:
```diff
@@ -446,6 +478,15 @@
+      - PsychiatristPDA
+      - TherapistPDA
+      - SocialWorkerPDA
+
```
unindent like this
```yml
loadouts:
- A
- B
- C
- D
```

alterae committed:
Merge branch 'master' into psychology-loadouts

alterae committed:
De-dent erroneously indented lines

I hate YAML. Not even sure how this particular mistake happened, but I
hate YAML.

Note to self: stop using Rider for YAML changes.

Lyndomen commented:
Under direction review, 24 hours

Lyndomen reviewed (approved):
Direction approved, our only feedback was that some of these options could be good to offer to other roles as well- but that's outside the scope of this PR :)

alterae commented:
> Direction approved, our only feedback was that some of these options could be good to offer to other roles as well- but that's outside the scope of this PR :)

So they could! I would love to hear specific suggestions over on the discord. 

But yea, out of scope for this. Thank you!

alterae commented:
@deltanedas this PR is now both deltanedas- and direction-approved, correct? can it become real now?
